config/environments/development.rb

```diff
@@ -64,4 +64,6 @@
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
 
   config.authentication_token = ENV.fetch('AUTHENTICATION_TOKEN', 'bats')
+
+  config.hosts << /[a-z0-9-.]+\.[a-z0-9-.]+\.development\.teacherservices\.cloud/
```

Comment:
Why do we need this?

Shouldn't this be using the review_aks.rb?

Reply:
This is for AKS deployments which all use production.rb. Can it go to production.rb ?

Reply:
I've removed this completely now.